src/ToolBox/SOS/lldbplugin/CMakeLists.txt

```diff
@@ -81,12 +81,15 @@ find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/lib/llvm-3.8/include")
 find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/lib/llvm-3.7/include")
 find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/lib/llvm-3.6/include")
 find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/lib/llvm-3.5/include")
+#FreeBSD
+find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/local/llvm39/include")
+find_path(LLDB_H "lldb/API/LLDB.h" PATHS "/usr/local/llvm38/include")
 
 if(LLDB_H STREQUAL LLDB_H-NOTFOUND)
     if(REQUIRE_LLDBPLUGIN)
         message(FATAL_ERROR "Cannot find LLDB.h. Try installing lldb-3.6-dev (or the appropriate package for your platform)")
     else()
-        message(WARNING "Cannot find LLDB.h Try installing lldb-3.6-dev (or the appropriate package for your platform)")
+        message(WARNING "Cannot find LLDB.h Try installing lldb-3.6-dev (or the appropriate package for your platformi). You may need to set LLVM_HOME if the build still can't find it.")
```

Comment:
A nit - typo - platformi -> platform. Also it seems that the same message should be displayed for the message above - and we should put the text to a common variable instead of duplicating it.